components/Header.tsx

```diff
@@ -36,10 +36,10 @@ const Header: React.FC = () => {
         <Box p={[2, 4, 5]} width={['100%', '100%', 2/3, 2/3]} textAlign="center">
           <Image src="/images/corona-virus-small.png" mb={0} title="CDC/ Alissa Eckert, MS; Dan Higgins, MAM / Public domain" alt="Kuvituskuva koronaviruksesta" borderWidth="0px" width="90px" />
         <Heading as="h1" color="rgb(51, 68, 85)" mb={[3, 3, 5]} mt={[3, 3, 2]} textAlign="center">{t('finland corona status')}</Heading>
-          <Text mb={5} fontFamily="Space Mono" color="rgb(51, 68, 85, 0.8)">Sivun tiedot pohjautuvat Helsingin Sanomien julkaisemaan <Link color="teal.500" href="https://github.com/HS-Datadesk/koronavirus-avoindata" isExternal>avoimeen dataan</Link> Suomen koronavirus-tartunnoista. HS on kerännyt aineiston julkisista lähteistä: tiedotustilaisuuksista, mediasta ja haastatteluista. Dataa päivitetään aina kun uusia tietoja tulee. Voit lukea lisätietoja koronaviruksesta alla olevista linkeistä:</Text>
+          <Text mb={5} fontFamily="Space Mono" color="rgb(51, 68, 85, 0.8)">{t('The data is based on')} <Link color="teal.500" href="https://github.com/HS-Datadesk/koronavirus-avoindata" isExternal>{t('open data')}</Link> {t('HS has gathered these data from')}</Text>
```

Comment:
I was just trying to figure out the interpolation of node-polyglot but this is good workaroung for now 👍

Reply:
I can make a tower with the corners that I cut 😆